### DISCUSSION TITLE
Cache for Open Source Projects

### DISCUSSION THREAD

**Discussion by dunklesToast** [vercel/turborepo#1105] [Help] [answered]
Hey,

I am curious if there is anything against using Turbo Cache for Open Source Projects so contributors can start developing more quickly. I've seen that it's possible to use [custom caching servers in the docs](https://turborepo.org/docs/features/remote-caching) (although I am not quite sure weather this is also controllable via `turbo.json`) but I wonder if there could be any downsides / security issues by having a global / open cache. The idea is that - with a custom solution - that everyone could read from the cache but only some users are allowed to upload (so it's not abused as cloud storage or so).

Would love to get some feedback on this :)

**Comment by anthonyshew** [comment, marked as answer]
My personal advice here would be to only allow trusted maintainers to share a cache. Publishing your cache credentials for anyone to use means that it could be abused in a variety of ways, including your Remote Cache being used for quite literally any project!

You are indeed able to use a custom cache with Turborepo, whether your source is open or closed!
